app/assets/stylesheets/shared/header.scss

```diff
@@ -257,12 +265,17 @@ header.navigation {
 
   .apply-button {
     @extend .button;
-    border-radius: 3px;
+    border-radius: 4px;
     display: inline;
     float: right;
-    font-weight: normal;
-    margin: 8px;
-    padding: 4px 16px;
+    font-weight: bold;
```

Comment:
Properties should be ordered border-radius, display, float, font-size, font-weight, margin, padding